controllers/namespace/namespace_controller.go

```diff
@@ -1,215 +0,0 @@
-package namespace
-
-import (
-	"context"
-	"fmt"
-	"os"
-	"strings"
-	"sync"
-
-	"github.com/go-logr/logr"
-	"github.com/red-hat-storage/ocs-operator/v4/controllers/util"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/runtime"
-	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/builder"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/event"
-	"sigs.k8s.io/controller-runtime/pkg/predicate"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-)
-
-// once is a pointer to the sync.Once type
-var once sync.Once
-
-// performCSIAddonsPodRestart a bool type to check if the csi-Addons pod should be restarted
-var performCSIAddonsPodRestart bool
-
-// NamespaceReconciler reconciles the namespaces starting with openshift-*
-// nolint:revive
-type NamespaceReconciler struct {
-	client.Client
-	ctx    context.Context
-	Log    logr.Logger
-	Scheme *runtime.Scheme
-}
-
-// +kubebuilder:rbac:groups=core,resources=pods,verbs=list;delete
```

Comment:
Checked, that removal is because of this.
We have access to pods in role.yaml [here](https://github.com/Nikhil-Ladha/ocs-operator/blob/skip-update-ns/config/rbac/role.yaml#L129)